node/node.go

```diff
@@ -144,11 +257,11 @@ func NewNode(ctx context.Context,
 	logger log.Logger,
 	options ...Option,
 ) (*Node, error) {
+
```

Comment:
Not really important, but can you revert the newline changes in the method, which have absolute no practical effect but will put this PR on the blame for those lines?